modules/filecheck/charts.go

```diff
@@ -0,0 +1,63 @@
+package filecheck
+
+import "github.com/netdata/go-orchestrator/module"
+
+var (
+	fileCharts = module.Charts{
+		fileExistenceChart.Copy(),
+		fileModTimeAgoChart.Copy(),
+		fileSizeChart.Copy(),
+	}
+
+	fileExistenceChart = module.Chart{
+		ID:    "file_existence",
+		Title: "File Existence (0: doesnt exist, 1: exists)",
```

Comment:
Title should be `0: doesn't exist`.

Reply:
I didn't add it deliberately. I guess it can break plugins.d parser. Let me check.

Reply:
because we wrap chart fields in `'`

https://github.com/netdata/go-orchestrator/blob/724cb40df90f21ef5d34af785146b1ecb7e145f5/pkg/netdataapi/api.go#L33-L34

Reply:
I got it, but I never saw `doesnt` in English literature. @joelhans is it possible to use `doesnt` ? Or should @ilyam8 to change for ` It does not`?

Reply:
@joelhans is it ok to use `exist/not exist` ?

Reply:
fixed, sql has [`exists/not exists`](https://vladmihalcea.com/sql-exists/)